cmd/notary/tuf.go

```diff
@@ -440,17 +448,36 @@ func (t *tufCommander) tufStatus(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if t.resetStatus {
+		return cl.Clear(t.archiveChangelist)
```

Comment:
should we have an interactive remove here?  It would match behavior with removing keys and delegation roles